src/lib/helpers.js

```diff
@@ -103,12 +103,12 @@ export function validateOrRaise(validatable, constraints) {
 
 export function formatDate(ISO8601DateString) {
   // http://momentjs.com/docs/#/displaying/
-  return moment(ISO8601DateString).utc().format('D MMM YYYY, HH:mm z');
+  return moment.utc(ISO8601DateString).format('D MMM YYYY, HH:mm z');
 }
 
 export function relativeDate(ISO8601DateString) {
   var formatedDate = formatDate(ISO8601DateString);
-  var relativeDate = moment(ISO8601DateString).utc().fromNow();
+  var relativeDate = moment(ISO8601DateString).fromNow();
 
```

Comment:
does this change affect other callers that expect `formatDate` return value to be in UTC?

Reply:
No it was wrong everywhere I think, this should fix it correctly. The thing is that moment was parsing the timestamp as if it were in the local timezone.

The change I did ensures `moment` thinks it is in UTC timezone, which all our timestamps are in.